### ISSUE TITLE
IEA license and terms of use

### ISSUE THREAD

**Issue by Ludee** [danielhuppmann/iea-netzero2050-datawrangler#1]
Great initiative @danielhuppmann 
I am highly concerned about the legal issues.
While IEA is providing the data online as xlsx there is no machine readable metadata and a suitable open data license.
`Source: IEA. All rights reserved` 
This means the data cannot be used and republished. Yet another example of bad Data Management and lacking understanding of licensing. They will not solve the climate crisis when they are not even able to share a dataset 😠

**Comment by danielhuppmann** [comment]
Hi @Ludee, indeed, that's why I specifically did not commit the data to the repository but only instruct users where to download the relevant files.

Per this Twitter discussion with the head of communication & digital office at IEA, reuse of the data is actually ok (even though this is inconsistent with the non-license statement) and they will attach a CC-BY license to the dataset.

-> https://twitter.com/jadmouawad/status/1394660439074787337

**Comment by danielhuppmann** [comment]
The IEA has now indeed released the data under a CC-BY-NC-SA license.

**Comment by Ludee** [comment]
CC-BY-NC-SA is not an open data license!

**Comment by danielhuppmann** [comment]
Correct, and I'm quite sure that I never said (here or on Twitter) that it is now "open data"...

Still, using a standard Creative Commons license is big step forward for them, compared to the 10 pages of legalese that applied to that dataset at the initial release (and that probably still applies to any other data that you can download from their website).
